doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,143 @@
+| `req.lcm.adp.03` | Automated deployment | The Architecture **should** support hitless upgrade of all software provided by the cloud provider that are not covered by `req.lcm.adp.02`. Whenever hitless upgrades are not feasible, attempt should be made to minimize the duration and nature of impact. |
+| `req.lcm.adp.04` | Automated deployment | The Architecture **must** support declarative specifications of hardware and software assets for automated deployment, configuration, maintenance and management. |
+| `req.lcm.adp.05` | Automated deployment | The Architecture **should** support automated process for Deployment and life-cycle management of CIM Instances. |
+| `req.lcm.cid.02` | CI/CD | The Architecture **should** support integration with CI/CD Toolchain for NFVI and CIM components Automation. |
+
+<p align="center"><b>Table 2-5:</b> Containerised Platform: LCM Requirements. </p>
+
+
+<a name="2.3.6"></a>
+### 2.3.6 Assurance
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.asr.mon.01` | Integration | The Architecture **must** include integration with various infrastructure components to support collection of telemetry for assurance monitoring and network intelligence. |
+| `req.asr.mon.02` | Monitoring | The Architecture **should** support Network Intelligence capabilities that allow richer diagnostic capabilities which take as input broader set of data across the network and from CNF workloads. |
+| `req.asr.mon.03` | Monitoring | The Architecture **must** allow for the collection and dissemination of performance and fault information. |
+| `req.asr.mon.04` | Network | The NFVI Network Fabric and Network Operating System **must** provide network operational visibility through alarming and streaming telemetry services for operational management, engineering planning, troubleshooting, and network performance optimisation. |
+
+<p align="center"><b>Table 2-6:</b> Containerised Platform: Assurance Requirements.</p>
+
+<a name="2.3.7"></a>
+### 2.3.7 Security
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.sec.gen.01` | General | The Architecture **must** provide workload isolation. |
+| `req.sec.gen.02` | General | The Architecture **must** support policy based RBAC. |
+| `req.sec.gen.03` | General | The Architecture **must** support a centralised authentication and authorisation mechanism. |
+| `req.sec.gen.04` | General | The Architecture **must** provide Operating System kernel isolation. |
```

Comment:
On reflection I think this is better as a "should" - there may be occasions where an operator may feel able to accept the risk due to other mitigations.  It still means we address it in the security chapter, but we don't mandate it.

Reply:
@tomkivlin made changes only to `req.sec.gen.01` and `req.sec.gen.04`